src/Altinn.App.Api/Controllers/DataController.cs

```diff
@@ -460,48 +519,76 @@ public async Task<ActionResult<DataPatchResponse>> PatchFormData(
             if (!InstanceIsActive(instance))
             {
                 return Conflict(
-                    $"Cannot update data element of archived or deleted instance {instanceOwnerPartyId}/{instanceGuid}"
+                    new ProblemDetails()
```

Comment:
I like that we use problem details more often, with the OTel setup this makes error responses a lot easier to debug 👏